backend/logistica_servicio/gestor_entregas/infraestructura/repositorio_entrega_camion.py

```
@@ -0,0 +1,104 @@
+            f'postgresql://{DB_USERNAME}:{DB_PASSWORD}@{DB_HOSTNAME}:{DB_PORT}/{DB_NAME}',
+            pool_size=5,
+            max_overflow=2,
+            pool_recycle=120,
+            pool_timeout=30,
+            pool_pre_ping=True,
+            connect_args={"connect_timeout": 10}
+        )
+
+# Use the dynamically determined engine
+engine = get_engine()
+Session = sessionmaker(bind=engine)
```

Ruff (0.8.2)

Redefinition of unused `Session` from line 1

(F811)